src/System.Console/src/System.Console.csproj

```diff
@@ -40,7 +42,14 @@
       <Link>Common\System\IO\EncodingHelper.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' != 'net46' ">
+  <!-- Windows : WinRT -->
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'netcore50' ">
+    <TargetingPackReference Include="Windows" />
```

Comment:
It appears as though you aren't actually using WinRT as you removed the WinRT package so you can remove this and the mscorlib.WinRt-Facade.csproj reference as well.